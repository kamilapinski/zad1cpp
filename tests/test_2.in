=1 1 1
USA 1
USA 2
Japan 2
-Brazil 3
monaco 4
=1 1 1
=1 1 0
=999999 999999 999999
=1 2 1000000
=9999999 5 10
=2344 5674321 10
Republic of Congo 0
=1 2 3
-Republic of Congo 1
-USA 1
USA 3
usa 2
Usa 1
UsA 2
=3 2 1
Republic of Congo 1
Republic of Congo 2
Republic of Congo 2
Republic of Congo 3
=1 2 3
-Japan 1
-Japan 2
Japan 2
UsA 3
Japan 3
Usa 2
=1 2 3
= 1 2 3
=1  2 3
=1 2  3
=1 2 3  
=1  2  3
=1 2 3