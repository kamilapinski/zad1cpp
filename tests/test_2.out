1. USA
2. Japan
1. USA
2. Japan
1. USA
2. Japan
3. Republic of Congo
1. USA
1. Usa
3. Japan
3. UsA
5. Republic of Congo
1. Republic of Congo
2. USA
3. Japan
3. UsA
5. Usa
1. Republic of Congo
2. Japan
2. USA
2. UsA
5. Usa
1. Republic of Congo
2. Japan
2. USA
2. UsA
5. Usa
